### PR TITLE
Fixed structs.write() when the argument was a byte array.

### PR DESCRIPTION
**Pull request by 0ssigeno**
Added debug print for struct, just for clarification.
Fixed lstrlenW behaviour when the argument were the argument was a pointer to bytes.

### CODE REVIEW

**Comment by xwings** [comment]
I guess some sort of test always healthy.

File: qiling/os/windows/structs.py [hunk 1]
```diff
@@ -293,6 +293,7 @@ def read(self, addr):
         raise NotImplementedError
 
     def generic_write(self, addr: int, attributes: list):
+        self.ql.dprint(D_INFO, "[+] Writing Windows object " + self.__class__.__name__)
```
Comment (xwings) [inline qiling/os/windows/structs.py:296]: all the content writing debug print we should move it to D_D_RPRT

**Comment by 0ssigeno** [comment]
Now gandcrab is executed with three different environment: 
administrator
user
administrator with russian keyboard

If the number of syscalls for each test is different, than we should be good, or at least, we have not broke something that is very important. For test each structure independently, i will make another PR

**Comment by xwings** [comment]
cool. thanks

i will merge for now

**Pull request merged**